core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -32,11 +32,14 @@
 
 	/** The static ball list. */
 	private ArrayList<Ball> ballStaticList = new ArrayList<Ball>();
-
-	/** The static ball dead list. */
+	
+	/**  The ball to be added to static List. */
```

Comment:
Yea, good spot. Will fix
I actually removed that list in my branch because i redesigned things. Then jeroen his graph used it intensively so I brought it back in the merge.